src/ploomber/cli/cloud.py

```diff
@@ -243,3 +244,44 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return _cloud_call
+
+
+def _get_input(text):
+    return input(text)
+
+
+def _email_input():
+    # Validate that's the first email registration
+    user_conf_path = Path(check_dir_exist(CONF_DIR), DEFAULT_USER_CONF)
+    conf = read_conf_file(user_conf_path)
+    email = conf.get('user_email', None)
+    if not email:
+        email = _get_input(
+            "Our users enjoy updates, support and funtivities "
+            "through email, please add your email would you like "
+            "to register (type email): ")
+        _email_validation(email)
```

Comment:
this will prompt the user every time, right? I think it should be a one-off thing

Reply:
No it'll be a one time off, check the _email_input func

Reply:
Can you add a test? I tried it and it prompted me twice in a row

Reply:
Added